### PR TITLE
chore: overhaul issue template

### PR DESCRIPTION
**Pull request by jdrueckert**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/crash-bug-report.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+name: Crash/Bug report
```
Comment (jdrueckert) [inline .github/ISSUE_TEMPLATE/crash-bug-report.md:2]:
```suggestion
name: 🐛 Crash/Bug Report
```

File: .github/ISSUE_TEMPLATE/improvement-request.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+name: Improvement request
```
Comment (jdrueckert) [inline .github/ISSUE_TEMPLATE/improvement-request.md:2]:
```suggestion
name: ✨ Improvement Request
```

File: .github/ISSUE_TEMPLATE/roadmap-draft.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+name: Roadmap draft
```
Comment (jdrueckert) [inline .github/ISSUE_TEMPLATE/roadmap-draft.md:2]:
```suggestion
name: 🛤️ Roadmap Draft
```

File: .github/ISSUE_TEMPLATE/documentation-request.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+name: Documentation request
```
Comment (jdrueckert) [inline .github/ISSUE_TEMPLATE/documentation-request.md:2]:
```suggestion
name: 📜 Documentation Request
```

**Review by jdrueckert — COMMENTED** [review]
Emojis like so maybe?

**Review by pollend — APPROVED** [review]

**Review by DarkWeird — APPROVED** [review]

**Pull request merged**